mobile/status.go

```diff
@@ -1071,6 +1072,17 @@ func writeHeapProfile(dataDir string) string { //nolint: deadcode
 	return makeJSONResponse(err)
 }
 
+// StartProfiling starts profiling and HTTP server for pprof
+func StartProfiling(address string) string {
```

Comment:
There is also `StartCPUProfile`, I am wondering how is this one different?

Reply:
Well it calls a different `pprof` function :D